aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/WebAuthnHelper.kt

```diff
@@ -75,22 +76,26 @@ internal class WebAuthnHelper(
     }
 
     suspend fun createCredential(requestJson: String, callingActivity: Activity): String {
-        try {
-            // Create the request for CredentialManager
-            val request = CreatePublicKeyCredentialRequest(requestJson)
-
-            // Create the credential
-            logger.verbose("Prompting user to create a PassKey")
-            val result: CreateCredentialResponse = credentialManager.createCredential(callingActivity, request)
-
-            // Extract the Public Key registration response. This is what we send to Cognito.
-            val publicKeyResult = result as? CreatePublicKeyCredentialResponse ?: throw WebAuthnFailedException(
-                "Android created wrong credential type",
-                AmplifyException.REPORT_BUG_TO_AWS_SUGGESTION
-            )
-            return publicKeyResult.registrationResponseJson
-        } catch (e: CreateCredentialException) {
-            throw e.toAuthException()
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            try {
+                // Create the request for CredentialManager
+                val request = CreatePublicKeyCredentialRequest(requestJson)
+
+                // Create the credential
+                logger.verbose("Prompting user to create a PassKey")
+                val result: CreateCredentialResponse = credentialManager.createCredential(callingActivity, request)
+
+                // Extract the Public Key registration response. This is what we send to Cognito.
+                val publicKeyResult = result as? CreatePublicKeyCredentialResponse ?: throw WebAuthnFailedException(
+                    "Android created wrong credential type",
+                    AmplifyException.REPORT_BUG_TO_AWS_SUGGESTION
+                )
+                return publicKeyResult.registrationResponseJson
+            } catch (e: CreateCredentialException) {
+                throw e.toAuthException()
+            }
+        } else {
+            throw WebAuthnNotSupportedException(cause = null)
```

Comment:
Should the message be updated to say its not supported because the Android version doesn't meet minimum requirements?